user/registration/src/main/java/in/projecteka/jataayu/registration/ui/activity/RegistrationActivity.kt

```diff
@@ -65,10 +71,18 @@ class RegistrationActivity : BaseActivity<RegistrationActivityBinding>() {
                     showErrorDialog(it.error.localizedMessage)
                 }
                 is PartialFailure -> {
-                    showAlertDialog(getString(R.string.failure), it.error?.message, getString(android.R.string.ok))
+                    val currentFragment = supportFragmentManager.findFragmentById(R.id.fragment_container);
```

Comment:
Check the error code for request exceeded and then based on that show the error

Reply:
Yes sure we will change that.